Launching-new-features.md

```diff
@@ -3,7 +3,8 @@ Please use the following process for launching major new features that lead to a
 - Before doing anything else, please:
     - Make sure e2e tests are present for your feature.
     - Make sure it is easy to toggle on and off, in case it needs to be aborted for the release. This implies proper gating of views, controllers, etc.
-- Fill out this [google form](https://forms.gle/m2u1VkQDXWee4euAA).
+    - Make sure that there is a clear set of CUJs for the feature listed in the [TDD](https://docs.google.com/document/d/1mnz8f708DZIa6BpUyRmbb0gCT6EKO3wIvWa_3rOEOYs/edit#bookmark=id.c1jeenukq729) doc for the feature.
+- Fill out this [google form](https://forms.gle/m2u1VkQDXWee4euAA). Please provide a link to the TDD in the form.
 - Nithesh (**@nithusha21**) will first do a quick audit of the feature, in order to ensure that it is ready for testing. If the feature is not ready, he will get back to you with what needs to be done, and abort the launch. If the feature passes initial checks, he will put you in touch with a product tester. (The expected turnaround time for this part is < 48 hours.)
```

Comment:
We should probably change this. Would you like to do it, or should we discuss with @vojtechjelinek?

Reply:
Added me for now.